### PR TITLE
feat: [META-966] - Document update for slow query logging

### PR DESCRIPTION
**Pull request by shaileshpandey**
Logger Documentation updated for 

- `db_slow_threshold_milliseconds`
- `GraphQL queries`

# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?
This change affects 1.5.1 and above releases.
 
It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: scst-store/logs.hbs.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ extensibility.
 
 If no verbosity level is specified when the Store is installed, the level is set to `default`.
 
+### <a id='slow-sql'></a> Slow SQL
+
+Slow sql statement will be logged only when verbosity level is set to trace and sql query took more than 200 milli seconds to execute. You can change this default by setting `db_slow_threshold_ms` value in values.yaml file and redeploy metadata store.
```
Comment (Manifaust) [inline scst-store/logs.hbs.md]: I wanted to ask does this configuration only apply to `amr` section of the values yaml?

Comment (shaileshpandey) [reply scst-store/logs.hbs.md]: I added it in AMR and Store both

Comment (Pivotal-Jeff-Jun) [inline scst-store/logs.hbs.md]: Can we highlight the word trace here? It's a key value that users can set it to right? 
Also does capitalization matter?

File: scst-store/logs.hbs.md [hunk 2]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
 * Some information in these payloads may be sensitive, and the user may not want them exposed in
 production environment logs.
 
+##  <a id='graphql-endptlog-out'></a> GraphQL endpoint log output (Only applicable to Artifactory Metadata Repository logs)
+
+When an GraphQL endpoint handles a request, the AMR generates two and five log lines. They are:
```
Comment (Pivotal-Jeff-Jun) [inline scst-store/logs.hbs.md]: do you mean `two to five logs lines.`?

File: scst-store/logs.hbs.md [hunk 3]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
...
+
+1. Every request will have received by endpoint will have `Processing request`. This will include operation called and actual fields asked in the response output 
+1. Every response will yield actual query and status
+1. If the endpoint returns a response body, it outputs a second `Request response` line with an extra
+key `payload`, and its value is set to the entire response body. This line is shown at the `debug`
+   verbosity level.
```
Comment (Pivotal-Jeff-Jun) [inline scst-store/logs.hbs.md]: we should either have all end with a period or with none. (im not too familiar what the style guide is)

File: scst-store/logs.hbs.md [hunk 4]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
...
+
+### <a id='graphql-endptlog-out-format'></a>  Format
+
+The logs use JSON output format.
```
Comment (Pivotal-Jeff-Jun) [inline scst-store/logs.hbs.md]: Should we say `The logs are in JSON format.`?

File: scst-store/logs.hbs.md [hunk 5]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
...
+
+When the AMR handles a request, it outputs some GraphQL endpoint access information in the following
+format:
+
+```console
+{"level":"info","ts":"2023-03-23T13:11:31.161531-06:00","logger":"Artifact Metadata Repository","msg":"Processing request","hostname":"shaileshp2DMD6R.vmware.com","getAllApps":"query getAllApps {\n  apps(latest:true) {\n    \n    timestamp\n    location {\n      alias\n    }\n  }\n}"}
+{"level":"info","ts":"2023-03-23T13:11:31.172953-06:00","logger":"Artifact Metadata Repository","msg":"Request response","hostname":"shaileshp2DMD6R.vmware.com","getAllApps":"query getAllApps {\n  apps(latest:true) {\n    \n    timestamp\n    location {\n      alias\n    }\n  }\n}","code":200,"response":"OK"}
```
Comment (Pivotal-Jeff-Jun) [inline scst-store/logs.hbs.md]: maybe we should use a dummy hostname. also I am not sure if you are okay with showing your name in the doc

**Review by Pivotal-Jeff-Jun — APPROVED** [review]

File: scst-store/logs.hbs.md [hunk 6]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
 * Some information in these payloads may be sensitive, and the user may not want them exposed in
 production environment logs.
 
+##  <a id='graphql-endptlog-out'></a> GraphQL endpoint log output (Only applicable to Artifactory Metadata Repository logs)
+
+When an GraphQL endpoint handles a request, the AMR generates two and five log lines. They are:
+
+1. Every request will have received by endpoint will have `Processing request`. This will include operation called and actual fields asked in the response output
```
Comment (julia-pu) [inline scst-store/logs.hbs.md]: I think the wording is a bit unclear. If I'm understanding it correctly, how about something like "Every request received will produce a `Processing request` log, which includes the name of the operation called and the fields that are being requested"

File: scst-store/logs.hbs.md [hunk 7]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
 * Some information in these payloads may be sensitive, and the user may not want them exposed in
 production environment logs.
 
+##  <a id='graphql-endptlog-out'></a> GraphQL endpoint log output (Only applicable to Artifactory Metadata Repository logs)
+
+When an GraphQL endpoint handles a request, the AMR generates two and five log lines. They are:
+
+1. Every request will have received by endpoint will have `Processing request`. This will include operation called and actual fields asked in the response output 
+1. Every response will yield actual query and status
```
Comment (julia-pu) [inline scst-store/logs.hbs.md]: My interpretation of the meaning is: "Every response will produce a log containing the actual query and the return status" is that correct?

File: scst-store/logs.hbs.md [hunk 8]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
...
+format:
+
+```console
+{"level":"info","ts":"2023-03-23T13:11:31.161531-06:00","logger":"Artifact Metadata Repository","msg":"Processing request","hostname":"shaileshp2DMD6R.vmware.com","getAllApps":"query getAllApps {\n  apps(latest:true) {\n    \n    timestamp\n    location {\n      alias\n    }\n  }\n}"}
+{"level":"info","ts":"2023-03-23T13:11:31.172953-06:00","logger":"Artifact Metadata Repository","msg":"Request response","hostname":"shaileshp2DMD6R.vmware.com","getAllApps":"query getAllApps {\n  apps(latest:true) {\n    \n    timestamp\n    location {\n      alias\n    }\n  }\n}","code":200,"response":"OK"}
+```
+
+####  <a id='graphql_key-val'></a> Key-value pairs
+
+Since JSON output format uses Key-value pairs, the tables in the following sections list each key and the meaning of their values.
```
Comment (julia-pu) [inline scst-store/logs.hbs.md]: Key doesn't need to be capitalized. Also maybe reword to something like "The following tables list the meaning of each key found in the logs"

File: scst-store/logs.hbs.md [hunk 9]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
...
+body payload data for both the request and response of an API call.
+
+The `debug` verbosity level, instead of the `default`, is used to display this information instead of `default`
+because:
+
+* Body payloads can be huge and some information in these payloads may be sensitive, and the user may not want them exposed in
+production environment logs.
+
+Logs containing payload information may be in below mentioned format
+format:
```
Comment (julia-pu) [inline scst-store/logs.hbs.md]: duplicate `format`?

File: scst-store/logs.hbs.md [hunk 10]
```diff
@@ -174,6 +178,90 @@ Moving the payload information at this level helps keep the production log outpu
...
+
+```console
+{"level":"info","ts":"2023-03-23T13:11:31.172966-06:00","logger":"Artifact Metadata Repository","msg":"Request response","hostname":"shaileshp2DMD6R.vmware.com","getAllApps":"query getAllApps {\n  apps(latest:true) {\n    \n    timestamp\n    location {\n      alias\n    }\n  }\n}","payload":{"apps":[{"timestamp":"2023-03-22T15:09:38.867371-06:00","location":{"alias":"1-Alias"}}]}}
+```
+
+##  <a id='slow_sql_query-out'></a> Slow SQL Query log output
+When verbosity level to `trace` you will see logs messages having some slow sql queries.
```
Comment (julia-pu) [inline scst-store/logs.hbs.md]: Small reword, "When verbosity level is set to `trace` you will see log messages containing slow sql queries"

File: scst-store/logs.hbs.md (hunk 1 above)
Comment (shaileshpandey) [reply scst-store/logs.hbs.md]: Made required change but can't do Capital since it is not correct verbosity Level

File: scst-store/logs.hbs.md (hunk 2 above)
Comment (shaileshpandey) [reply scst-store/logs.hbs.md]: Good catch. Fixed

**Review by julia-pu — APPROVED** [review]
